backend/alembic/versions/0024_sibling_roms_db_view.py

```diff
@@ -22,32 +22,35 @@ def upgrade() -> None:
         batch_op.create_index("idx_roms_moby_id", ["moby_id"])
 
     connection = op.get_bind()
+    null_safe_equal_operator = (
+        "IS NOT DISTINCT FROM" if connection.engine.name == "postgresql" else "<=>"
+    )
 
     connection.execute(
         sa.text(
-            """
+            f"""
             CREATE VIEW sibling_roms AS
             SELECT
                 r1.id AS rom_id,
                 r2.id AS sibling_rom_id,
                 r1.platform_id AS platform_id,
                 NOW() AS created_at,
                 NOW() AS updated_at,
-                CASE WHEN r1.igdb_id <=> r2.igdb_id THEN r1.igdb_id END AS igdb_id,
-                CASE WHEN r1.moby_id <=> r2.moby_id THEN r1.moby_id END AS moby_id
+                CASE WHEN r1.igdb_id {null_safe_equal_operator} r2.igdb_id THEN r1.igdb_id END AS igdb_id,
+                CASE WHEN r1.moby_id {null_safe_equal_operator} r2.moby_id THEN r1.moby_id END AS moby_id
             FROM
                 roms r1
-            JOIN 
-                roms r2 
-            ON 
+            JOIN
+                roms r2
+            ON
                 r1.platform_id = r2.platform_id
                 AND r1.id != r2.id
                 AND (
-                    (r1.igdb_id = r2.igdb_id AND r1.igdb_id IS NOT NULL AND r1.igdb_id != '')
```

Comment:
`r1.igdb_id != ''` comparison is not needed, as the `igdb_id` column is Integer. This doesn't fail on MySQL/MariaDB, but PostgreSQL complain because of `''` not being a possible value for the column.